css/styles.css

```diff
@@ -0,0 +1,60 @@
+
+body {
+  max-width: 320px;
+  margin: 0 auto;
+}
```

Comment:
|  Your Design  |Figma Design |
| :---:   | :---: | 
| <img width="610" alt="Screenshot 2022-11-30 at 22 55 21" src="https://user-images.githubusercontent.com/99246359/204916664-349f107c-cd15-4dd4-85b5-7be7f4abad45.png"> |<img width="402" alt="Screenshot 2022-11-30 at 22 58 51" src="https://user-images.githubusercontent.com/99246359/204917189-b049d987-9918-4f19-bedf-021fce93fd92.png">  |

- I see you have manually added a fixed width value to lines 27, 37 and 45. I suggest that you kindly replace these lines with `width: 100%` 🥂